cms/templates/cms/home_page.html

```diff
@@ -111,26 +111,30 @@
     </section>
   </div>
 
-
   <footer>
-    <div class="container">
-      <div class="row">
-        <div class="col-md-8">
-          <img src="static/images/logo-mit-white.png" alt="">
-          <div class="footer-links"><a href="add" target="_blank">About MicroMaster’s</a> <a href="https://www.edx.org/" target="_blank">edX</a> <a href="https://odl.mit.edu/" target="_blank">Office of Digital Learning</a>
-          </div>
-          <div class="footer-address">
-            Massachusetts Institute of Technology<br> Cambridge, MA 02139
-          </div>
-        </div>
-        <div class="col-md-4 text-right">
-          <div class="footer-cta"></div>
-          <div class="footer-copy">&copy; 2016, Massachusetts Institute of Technology</div>
+     <div class="container">
```

Comment:
definitely not necessary for MVP. i was just asking. i'll make a note of this and maybe open an issue